src/test/fuzz/tx_pool.cpp

```diff
@@ -230,7 +230,8 @@ FUZZ_TARGET_INIT(tx_pool_standard, initialize_tx_pool)
         Assert(it->second.m_result_type == MempoolAcceptResult::ResultType::VALID ||
                it->second.m_result_type == MempoolAcceptResult::ResultType::INVALID);
 
-        const auto res = WITH_LOCK(::cs_main, return AcceptToMemoryPool(chainstate, tx_pool, tx, bypass_limits));
+        CChainState& chainstate{node.chainman->ActiveChainstate()};
+        const auto res = WITH_LOCK(::cs_main, return AcceptToMemoryPool(chainstate.m_params, tx_pool, chainstate, tx, GetTime(), bypass_limits, /* test_accept= */false));
```

Comment:
Its seems odd to change this to:

```
        const auto res = WITH_LOCK(::cs_main, return AcceptToMemoryPool(::Params(), tx_pool, chainstate, tx, GetTime(), bypass_limits, /* test_accept= */false));
```

in the first commit, and then update it again to use `chainstate.m_params` in the second commit. You can just go straight to using `chainstate.m_params` in the first commit.

Reply:
Agreed. Changed it to the first commit.